tests/unit/PaymentErrorDialog.spec.ts

```diff
@@ -0,0 +1,83 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { getVuexStore } from '@/store'
+import { shallowMount, mount } from '@vue/test-utils'
+import { PaymentErrorDialog } from '@/components/dialogs'
+import ErrorContact from '@/components/common/ErrorContact.vue'
+
+Vue.use(Vuetify)
+
+const vuetify = new Vuetify({})
+const store = getVuexStore()
+
+// Prevent the warning "[Vuetify] Unable to locate target [data-app]"
+document.body.setAttribute('data-app', 'true')
```

Comment:
NOTE!!!
This code is needed for the dialog when it is **mounted** but it will cause errors for the dialog when it is **shallow mounted**, so be advised. (When shallow mounted, wrapper.html() returns an empty body.)

Reply:
I've seen this before as well, specifically when trying to reach into a Vuetify sub component, like a `v-menu` for example. This is interesting, thanks for pointing this out!